test/morphology/test_skeletonize.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+from kornia.morphology import skeletonize
+from kornia.testing import assert_close
+
+class TestSkeletonize:
```

Comment:
```suggestion
class TestSkeletonize(BaseTester):
```
ideally use the common api for testing